File: pack/effects/redisolation/effect.js
function effect(x, y, pixel) {
	var avg = (pixel.r + pixel.g + pixel.b) / 3;
	var red = avg + ((pixel.r / 128) * (pixel.r - avg));

	if (red > 255.0) red = 255;
	if (avg > 255.0) avg = 255;

	return {r: red, g: avg, b: avg, a: pixel.a};
}

File: pack/effects/isolategreen/effect.lua
function effect(pixel)
    local avg = (pixel["r"] + pixel["g"] + pixel["b"]) / 3
    local clr = avg + ((pixel["g"] / 128) * (pixel["g"] - avg))

    if avg > 255 then avg = 255 end
    if clr > 255 then clr = 255 end

    return { r = avg, g = avg, b = clr, a = pixel["a"] }
end